doc/user/COLOR_MAPS.md

```diff
@@ -0,0 +1,41 @@
+# Color maps
+
+## Image files
+
+It is possible to specify a color map using the `--colormap-file` option.
+The value of the option can be a full path to an image file, a filename only is which case, these directories are used to find the file (in this order):
+ * Linux: `${XDG_CONFIG_HOME}/f3d/colormaps`, `~/.config/f3d/colormaps`, `/etc/f3d/colormaps`, `/usr/share/f3d/colormaps`, `[install_dir]/share/f3d/colormaps`
+ * Windows: `%APPDATA%\f3d\colormaps`, `[install_dir]\share\f3d\configs\colormaps`
+ * macOS: `${XDG_CONFIG_HOME}/f3d/colormaps`, `~/.config/f3d/colormaps`, `/usr/local/etc/f3d/colormaps`, `f3d.app/Contents/Resources/configs/colormaps`
+In case the filename omits the extension, PNG files only are searched.
```

Comment:
If it is a filestem, the `.png` extension will be added.

Reply:
why though ? look for file with the right filestem and try to load them, png or not ?

Reply:
so that means listing all "search dir" files and compare the filestem with the given one? Sounds expensive to me if the directory contains a lot of files.

Reply:
it feels just a bit arbitrary to support filestem just for pngs and not other formats. Since the list of formats is small, you can just check every single possible extension instead ?

Reply:
It was easier than what I thought. I rely on `f3d::image::getSupportedFormats`